modules/ob-configuring-pods-in-build.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// * configuring/configuring-openshift-builds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-configuring-pods-in-build_{context}"]
+= Pod configuration
+
+You can use the following optional `Build` CR fields to configure {builds-shortname} pods:
+
+* The `spec.tolerations` field specifies the tolerations for the {builds-shortname} pod with the following limitations:
++
+** Only the `NoSchedule` taint `effect` is supported
```

Comment:
```suggestion
** Only the `NoSchedule` taint `effect` is supported.
```
If possible, include the correct object reference for `effect`. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references.